tests/workloads/megatron_run/test_megatron_run.py

```
@@ -56,34 +57,30 @@ def test_missing_slurm_metadata_fails(self, base_tr: TestRun) -> None:
+                "throughput per GPU (TFLOP/s/GPU): 494.6 |\n",
+                False,
+            ),
+            (
+                "0:0",
+                "[2026-01-16 07:32:39] iteration 6/100 | elapsed time per iteration (ms): 15639.0 | "
+                "throughput per GPU (TFLOP/s/GPU): 494.6 |\n",
+                True,
+            ),
+            ("0:0", "validation loss at iteration 1.0", True),
+            ("15:0", "validation loss at iteration 1.0", False),
+        ),
```

Ruff (0.15.9)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)